include/Token.h

```diff
@@ -571,6 +571,7 @@ enum class TokenKind : uint16_t {
     MacroQuote,
     MacroEscapedQuote,
     MacroPaste,
+    DummyMacroPaste,
```

Comment:
Reading through this I figured out that "DummyMacroPaste" indicates a macropaste with spacing so that it doesn't really do any useful pasting; I feel like somewhere in this change a commend should spell that out.